components/search/RegionSearch/RegionSelect.tsx

```diff
@@ -78,6 +83,7 @@ const RegionSelect: React.FC<RegionSelectProps> = ({
 
   return (
     <div>
+      <LocationCombobox searchLocations={handleFindRegion} setRegionId={onRegionSelect} />
```

Comment:
`LocationCombobox` is missing a crucial prop. It has no way of retrieve the API's result from `RegionSelect`. Add a new prop called `regionsArray`. This will pass the API data onto `LocationCombobox`.

```suggestion
      <LocationCombobox searchLocations={handleFindRegion} setRegionId={onRegionSelect} regionsArray={regionSearchOutput} />
```